metrics/report/README.md

```diff
@@ -0,0 +1,12 @@
+# PnP failure Report tool
+
+This tool reports by email the obtained PnP results from checktool
```

Comment:
What is "checktool" (and if it's a command, it should be rendered like: `checktool`).